unskript-ctl/unskript-client.py

```diff
@@ -1359,6 +1362,48 @@ def list_creds():
         index += 1
     print(tabulate(creds_data, headers='firstrow', tablefmt='fancy_grid'))
     
+def start_debug():
+    """start_debug Starts Debug session. This assumes that the remote
+       config is copied to /tmp as  /tmp/remoteconfig. If this file
+       does not exist, then this function will raise an exception.
+    """
+    if os.path.exists("/tmp/remoteconfig") is False:
+        raise Exception("Required Remote Configuration not present. Ensure /tmp/remoteconfig file is present.")
+
+    command = ["openvpn --config /tmp/remoteconfig"]
+    process = subprocess.Popen(command,
+                               stdout=subprocess.PIPE,
+                               stderr=subprocess.STDOUT,
+                               shell=True)
+
```

Comment:
What if the file exists, but is corrupt. Will the below checks detect this correctly?
Should we look for presence/status of the tunnel interface instead to declare victory ?

Reply:
Yes, If the file is corrupt, Openvpn Process will raise error, and return-code will be non-zero.  Added check to ensure tun interface is created. Only when the tunnel is Up, the interface is created.